slam/mapping.py

```diff
@@ -126,8 +85,13 @@ def spherical_mapping(mesh, mapping_type='laplacian_eigenvectors',
 def disk_conformal_mapping(mesh, lap_type='conformal',
                            boundary=None, boundary_coords=None):
     """
-    compute comformal mapping of a mesh to a disk
-    ADD ref
+    compute comformal mapping of a mesh to a disk, see the following references:
```

Comment:
Computes

Reply:
ok, well spotted!
Let me try to update the PR with only that change

Reply:
Nice it worked apparently